gradle.properties

```diff
@@ -1 +1,3 @@
+org.gradle.jvmargs=-Xmx8g
```

Comment:
This is useful if running, for example, OpenRewrite recipes which need lots of resources to parse the project.

Reply:
Hmm good point! It could have downsides on more constrained environments. Let me remove this.

Reply:
Removed